samples/msal-node-samples/ElectronSystemBrowserTestApp/public/successTemplate.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="Content-Security-Policy" content="script-src 'self'" />
+    <title>Electron system browser auth redirect page</title>
+    <style>
+    .container {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+        padding: 10px;
+        font-size: 20px;
+     }
+    </style>
+</head>
+<body>
+    <div class='container'>
+        <h5>You are signed in now, press Continue to open your app</h5>
```

Comment:
It's still not really clear to me why the user needs to interact with this page at all. Can you explain?

Reply:
Good point. I removed the link and redirected the user to the application on page load.

Reply:
Sounds good. Should we change this to say something like this then?
```suggestion
        <h5>You are now signed in. You may close this window.</h5>
```